netmanager/src/views/components/DataDisplay/Map/MapFilter.js

```diff
@@ -18,19 +18,28 @@ import "assets/css/map-filter.css";
 
 const FilterIcon = ({ fill, stroke }) => {
   return (
-    <svg
-      xmlns="http://www.w3.org/2000/svg"
-      height="24"
-      viewBox="0 0 24 24"
-      width="24"
-    >
-      <path d="M0 0h24v24H0z" fill="none" />
+    <svg xmlns="http://www.w3.org/2000/svg"
+      width="22" height="22" fill="#2f67e2"
+      class="bi bi-funnel-fill"
+      viewBox="0 0 16 16">
+      <path d="M1.5 1.5A.5.5 0 0 1 2 1h12a.5.5 0 0 1 .5.5v2a.5.5 0 0 1-.128.334L10 8.692V13.5a.5.5 0 0 1-.342.474l-3 1A.5.5 0 0 1 6 14.5V8.692L1.628 3.834A.5.5 0 0 1 1.5 3.5v-2z" />
       <path
-        d="M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z"
-        stroke={stroke || "white"}
-        fill={fill || "white"}
-      />
+      stroke={stroke || "white"}
+      fill={fill || "white"}/>
     </svg>
+    // <svg
+    //   xmlns="http://www.w3.org/2000/svg"
+    //   height="24"
```

Comment:
We should all avoid adding commented code to our repo. Unless it is really really necessary. Thanks.